src/plugins/console/public/lib/utils/index.ts

```diff
@@ -127,6 +127,11 @@ export const replaceVariables = (
   const bodyRegexTripleQuotes = /([\\"]?)"""\${(\w+)}"""(?!")/g;
 
   return requests.map((req) => {
+    // safeguard - caller passes any[] from editor's getRequestsInRange() to requests
```

Comment:
The `any[]` come from:
- https://github.com/elastic/kibana/blob/v8.9.0/src/plugins/console/public/application/models/sense_editor/sense_editor.ts#L290-L293
- https://github.com/elastic/kibana/blob/v8.9.0/src/plugins/console/public/application/hooks/use_send_current_request/use_send_current_request.ts#L35-L36
- https://github.com/elastic/kibana/blob/v8.9.0/src/plugins/console/public/application/models/sense_editor/sense_editor.ts#L478-L479